### ISSUE TITLE
Specs with the same suite + spec name get mis-reported in Eclipse

### ISSUE THREAD

**Issue by greghaskins** [greghaskins/spectrum#22]
May not be Eclipse-specific, but specs with the same JUnit description `spec name(suite name)` get coalesced by Eclipse. This is probably a limitation of JUnit's `Description` and Eclipse's runner that Spectrum will need to work around. Related to #3 .

For example:

``` java
@RunWith(Spectrum.class)
public class WeirdSpecs {{

    describe("a thing", () -> {
        it("should do something", () -> {
            assertThat("failing", false);
        });
    });

    describe("a thing", () -> {
        it("should do something", () -> {
            assertThat("passing", true);
        });
    });

}}
```

The above gets reported as simultaneously passing, failing, _and_ being incomplete:

<img width="323" alt="screen shot 2016-02-01 at 1 33 02 am" src="https://cloud.githubusercontent.com/assets/285310/12710357/d9c30dd6-c883-11e5-91c8-eb59577cc5d6.png">

**Comment by greghaskins** [comment]
Resolved by #65

**Comment by bvkatwijk** [comment]
Still running into this issue (v1.2.0) when another layer is added: 
```java
cribe("wrapper 1", () -> {
	describe("a thing", () -> {
		it("should do something", () -> {
			Assert.fail();
		});
	});
});
describe("wrapper 2", () -> {
	describe("a thing", () -> {
		it("should do something", () -> {
			//
		});
	});
});
```
![screen shot 2018-09-24 at 14 58 03](https://user-images.githubusercontent.com/18490578/45953447-687c9080-c00a-11e8-8486-a8a6b2097965.png)
